server/integration-test/src/test/java/com/oceanbase/odc/service/session/ConnectConsoleServiceTest.java

```diff
@@ -78,21 +83,300 @@
  */
 public class ConnectConsoleServiceTest extends ServiceTestEnv {
 
-    private final String sessionid = "10000";
+    private final String sessionId = "10000";
     @MockBean
     private ConnectSessionService sessionService;
     @Autowired
     private ConnectConsoleService consoleService;
     @Autowired
     private DefaultDBSessionManage defaultConnectSessionManage;
 
+    @Test
+    public void editProcedureForOBMysql_normal_successResult() throws Exception {
```

Comment:
it is not unit testing, it is implemenation instead.
should add DBPLModifyHelperTest instead, we may always avoid write logic in test cases.